src/Compilers/CSharp/Portable/CSharpParseOptions.cs

```diff
@@ -86,6 +86,7 @@ public override IEnumerable<string> PreprocessorSymbolNames
             kind: other.Kind,
             preprocessorSymbols: other.PreprocessorSymbols)
         {
+            _features = other._features;
```

Comment:
It feels like we should simply pass ```other._features``` as the last argument in the constructor initializer.